docs/user-guide/home/global-modules/README.md

```diff
@@ -0,0 +1,134 @@
+
+> It is possible that some of the global modules are not displayed for you. This will be the case if said module is not activated in the [Project settings](../../projects/project-settings/) in any of the active projects in you instance. If for example **News** module is not enabled for any of the active projects, you will not see it in the **Global modules** menu. 
+
+## Projects
+
+**Projects** global module will display a list of the projects that exist in your OpenProject instance, which you are a member of and/or have the right to see (for example as an administrator) and public projects.
+
+You can create a new project here directly by using the  **+ Project** button at the bottom of the left side menu.
+
+![OpenProject projects overview in the global modules menu](openproject_global_modules_projects.png)
+
+On the left you will have the following options:
+
+**All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects** for you.
+
+**My projects** is the view that will show projects you are a member of and public projects. When this menu entry is selected, an **I am member** filter will be activated under Filters. You can then add various other filters to adjust the view.
+
+![OpenProject global modules my projects](openproject_global_modules_myprojects_filter.png)
+
+**Public projects** will list all projects that have been set to be public.
+
+**Archived projects**  will list all archived projects that you were a member of or have the right to see.
+
+>Note: you can also navigate to the **Projects** through [**Select a project** dropdown menu](https://www.openproject.org/docs/user-guide/projects/#projects-list ) and by clicking on **View all projects** button in the **Projects** block on the home page.
+
+## Activity
+
+**Activity**  global module provide an overview of all project activities across the projects you are a member of, have the rights to see and public projects. 
+
+![OpenProject_user_guide_global_module_activity](openproject_global_modules_activity.png)
+
+The timeframe for tracing and displaying activities starts with the current date. You can adjust how far back the activities should be traced by adjusting [System settings](../../../system-admin-guide/system-settings/general-settings/).
+
+You can adjust the view by selecting respective filters on the left and clicking the **Apply** button. 
+
+>Note: **Changesets** filter comes from repositories that are managed by OpenProject. For example, if you make a commit to a GIT or SVN repository, these changes will be displayed here. At the moment this filter is only relevant for self-hosted editions.
+
+
+
+## Work packages
+
+**Work packages** global module will show a list of all the work packages from the projects you are a member of and all public projects. You can select your **Favorite** and **Default** work package filters in the left side menu.
+
+PUBLIC - ???
```

Comment:
The work package module work the same as inside a project. You could have work package table views marked as Public or Private. Note that if a view is marked as Private or Public but also Favourite they will be shown in the Favourite section.